src/ui/Base/Component.ts

```diff
@@ -425,6 +425,9 @@ export class ComponentEvents {
         } else if (args && JQueryUtils.isInstanceOfJqueryEvent(args[0])) {
           if (args[1] != undefined) {
             args = [args[1]];
+          } else if (args[0].hasOwnProperty('originalEvent')) {
+            // If the event is a jQuery Event but no args are found, let's use the ones of the originalEvent, if any.
+            args = [args[0].originalEvent];
```

Comment:
Would it be possible to unit test this change?

Also, does the `originalEvent` property hold the event args? Or should it be the `.originalEvent.detail`?

Reply:
The detail are not enough, for example if you wrap a keyboard event, detail is pretty dumb/empty and you'll be missing the pressed key, for example

Reply:
Yes UTs are possible